std/socket.d

```diff
@@ -3522,8 +3522,8 @@ class UdpSocket: Socket
             @property @trusted Address localAddress() { assert(0); }
             @trusted ptrdiff_t send(const(void)[] buf, SocketFlags flags) { assert(0); }
             @safe ptrdiff_t send(const(void)[] buf) { assert(0); }
-            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, Address to) { assert(0); }
-            @safe ptrdiff_t sendTo(const(void)[] buf, Address to) { assert(0); }
+            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, in Address to) { assert(0); }
+            @safe ptrdiff_t sendTo(const(void)[] buf, in Address to) { assert(0); }
```

Comment:
I think it would be possible to add const and non-const overloads, and have the non-const version call the const version; however, this creates possible problems such as one part of the code overrriding the old non-const overload, but another part of the code call the const overload, which is not going to call the overriding method.